src/k2/httpproxy/HTTPProxy.cpp

```diff
@@ -528,22 +550,17 @@ void HTTPProxy::_registerMetrics() {
     _metric_groups.clear();
     std::vector<sm::label_instance> labels;
     labels.push_back(sm::label_instance("total_cores", seastar::smp::count));
```

Comment:
i don't think we need this label. I'd just remove it. Same for the k23si library

Reply:
Removed. k23si  doesn't have label.